.eslintrc.js

```diff
@@ -0,0 +1,46 @@
+const fs = require('fs');
+const path = require('path');
+
+const prettierOptions = JSON.parse(fs.readFileSync(path.resolve(__dirname, '.prettierrc'), 'utf8'));
+
+module.exports = {
+  extends: ['airbnb-typescript/base', 'plugin:@typescript-eslint/recommended', 'prettier'],
+  plugins: ['@typescript-eslint', 'prettier'],
+  parser: '@typescript-eslint/parser',
+  env: {
+    node: true,
+    es6: true,
+  },
+  parserOptions: {
+    project: './tsconfig.json',
+    ecmaVersion: 2019,
+  },
+  rules: {
```

Comment:
Here is my usual eslint typescript setup. Without prettier and without importing some other config:
https://gist.github.com/davidlkang/baae72a59da1a03a6040488a412b1f72

I think we should have a chat on this.